app/controllers/admin/items_controller.rb

```diff
@@ -0,0 +1,70 @@
+class Admin::ItemsController < Admin::BaseController
+  def index
+    @merchant = Merchant.find(params[:merchant_id])
+  end
+
+  def update
+    @item = Item.find(params[:item_id])
+    if params[:commit] == "Update Item"
+      @item.update(item_params)
+      save_process(@item, render_arg = :edit, message = "#{@item.name} is updated.")
+    else
+      activation_process(@item)
+    end
```

Comment:
Update is the action used to toggle the active status and to update an item's attributes. The conditional filters which process based on the presence of the Update Item commit param.